tests/conftest.py

```diff
@@ -243,9 +243,9 @@ def __init__(self, tmpdir, env):
         autostart.chmod(0o755)
         bin_path = os.path.join(BINDIR, 'herbstluftwm')
         self.proc = subprocess.Popen([bin_path, '--verbose'], env=env,
-                bufsize=0,  # essential for reading output with selectors!
-                stdout=subprocess.PIPE,
-                stderr=subprocess.PIPE)
+                                     bufsize=0,  # essential for reading output with selectors!
+                                     stdout=subprocess.PIPE,
+                                     stderr=subprocess.PIPE)
```

Comment:
Here, the indentation was "wrong" because I found the line length more important than the alignment of the parameters below each other. How is this handled in python generally? Is the following a solution solving both "violations"?

    self.proc = subprocess.Popen(
        [bin_path, '--verbose'], env=env,
        bufsize=0,  # essential for reading output with selectors!
        stdout=subprocess.PIPE,
        stderr=subprocess.PIPE)

Reply:
True, your suggestion is a valid solution for that check.
I updated the PR

Reply:
According to [pep8](https://www.python.org/dev/peps/pep-0008/#indentation) (which most projects follow), these two possibilities are fine:

- Aligning with `(` (with content on the line containing the `(`)
- Hanging indent (indented by four spaces, without content on the line containing the `(`)

So yup, your proposal is fine.